mobile/mobile-core/device-discovery.ts

```diff
@@ -144,7 +144,10 @@ class IOSDeviceDiscoveryStub extends DeviceDiscovery {
 	}
 
 	public startLookingForDevices(): IFuture<void> {
-		this.$logger.warn(this.error);
+		if(this.error) {
```

Comment:
Maybe we need to print something like `ios releated commands are not supported on Linux` if the os isLinux